APKBUILD.5

```diff
@@ -0,0 +1,488 @@
+
+.It Cm provides
+Specifies that the package "provides" the same contents as another package.
+This is typically used for virtual dependencies.
+
+.It Cm provider_priority
+Specifies the numeric value for
+.Xr apk 8
+to use for the package when considering which provider should be installed for
+the same
+.Cm provides
+virtual provider.
+
+.It Cm replaces
+Specifies packages that the package replaces.  This is typically used for
+packages renamed by upstream.
+
+.It Cm triggers
+Specifies a trigger script used by the package.  A trigger script is a shell
+script that is called whenever monitored files or directories are modified.
+You may specify the paths to monitor using the triggers variable as follows:
+
+.Li Dq $pkgname.trigger=/usr/share/man:/usr/local/share/man
+
+This will run the package trigger script whenever files in
+.Pa /usr/share/man
+or
+.Pa /usr/local/share/man
+are created, modified, or removed.
+
+.El
+
+
+.Ss options
+The
+.Cm options
+variable allows you to set parameters for the package at build time.  There are
+a number of valid options you may set, and you may set multiple options by
+inserting a space between each one.
+
+.Bl -tag -width Ds
+.It Cm !archcheck
+Specifies that the package contains binaries that cannot run on the target
+architecture.  This is primarily used for packages containing firmware, and
+should typically never need to be used.
+
+.It Cm charset.alias
+Specifies that the package ships a /usr/lib/charset.alias file and that it
+should be installed on the user's system.  This is almost never the case.  Do
+not use this option.
+
+.It Cm !check
+Specifies that the package will not run a test suite.  The reason for disabling
+the check phase should be noted in a comment.
+
+.It Cm !checkroot
+Specifies that the package's test suite will be run as a non-privileged user
+instead of using
+.Xr fakeroot 8 .
+This is necessary for some test suites which fail when run as root.
+
+.It Cm !dbg
+(Adélie-specific) Specifies that the package should not be built with a debug
+information package.  This is typically used on packages that do not generate
+debug information (such as pure Python packages) or packages that do not
+support debug information packages (such as LLVM).
+
+.It Cm !fhs
+Specifies that the package violates FHS and installs to a location such as
+.Pa /usr/local ,
+.Pa /opt ,
+or
+.Pa /srv .
+
+.It Cm ldpath-recursive
+Specifies that
+.Xr abuild 1
+should use the
+.Fl --recursive
+argument to
+.Xr scanelf 1
+when attempting to find shared library (.so) dependencies for the package.
+
+.It Cm !libc_eglibc
+Specifies that the package cannot be built on the eglibc libc.
+
+.It Cm !libc_musl
+Specifies that the package cannot be built on the musl libc.
+
+.It Cm !libc_uclibc
+Specifies that the package cannot be built on the uclibc libc.
+
+.It Cm libtool
+Specifies that the package requires its libtool (.la) files.  They will not be
+automatically removed by
+.Xr abuild 1 .
+
+.It Cm net
```

Comment:
abuild doesn't implement network restrictions (yet). It's planned feature, but documentation should reflect the current state...

Reply:
[`rootbld` does](https://github.com/alpinelinux/abuild/blob/master/abuild.in#L2169).